modules/docs/arrow-docs/docs/_config.yml

```diff
@@ -1,10 +1,3 @@
-defaults:
-  -
```

Comment:
removing this breaks the api docs

Reply:
Indeed, will fix it in the next commit too, thanks for noticing it :+1: